src/styles/less/ParentFooter.less

```diff
@@ -1,43 +1,41 @@
 /* Footer Styles */
 
 .no-scroll {
-	position: absolute;
-	bottom: 0;
-	width: 100%;
+  position: absolute;
+  bottom: 0;
+  width: 100%;
 }
 
 footer {
-	// position: absolute;
-	// bottom: 0;
-	// width: 100%;
-
-	font-size: 0.89rem;
+  // position: absolute;
+  // bottom: 0;
+  // width: 100%;
+
+  font-size: 0.89rem;
+  background: #6a7179;
+  color: #fbfbfb;
+
+  .inner-container {
+    display: flex;
+    justify-content: space-evenly;
+  }
+
+  .parent-footer-nav-container {
+    width: 10%;
+    display: flex;
+    justify-content: space-around;
+  }
+
+  .parent-footer-nav-div-link {
+    text-decoration: underline;
+    color: #fbfbfb;
+  }
 
-    background: #6A7179;
+  a {
     color: #fbfbfb;
-     padding: 2rem 30vh;
-    .inner-container {
-      display: flex;
-	  justify-content: center;
-      justify-content:space-between;
-    };
-
-    .parent-footer-nav-container {
-      display: flex;
-	  justify-content: space-around;
-    }
-
-    .parent-footer-nav-div-link {
-      margin: 0 8%;
-      text-decoration: underline;
-      color: #fbfbfb;
-    }
-
-    a {
-      color: #fbfbfb;
-    }
-
-    a:hover {
-      color: rgb(102,154,249);
-    }
-  };
\ No newline at end of file
+  }
+
+  a:hover {
+    color: rgb(102, 154, 249);
+  }
+}
```

Comment:
Is this the closing tag for the footer styling, and if so, why not put it before the inner container styling on line 18? If not, where is the closing tag?

Reply:
Yes, it is the closing tag for the footer styling.

Reply:
Ok. I'm not sure if it matters, but why is it all the way down there? Is there a reason to include all the other stying inside the footer bracket?

Reply:
Just checked putting the closing tag, wouldn't make any difference. We can use either way I believe.

Reply:
Ok then.